pubspec.yaml

```diff
@@ -4,10 +4,10 @@ version: 0.14.10-dev
 homepage: https://github.com/dart-lang/pana
 
 environment:
-  sdk: '>=2.8.0 <3.0.0'
+  sdk: '>=2.10.4 <3.0.0'
```

Comment:
`pub.dev` is running on `2.10.0`. Can we have 2.10.0 as minimum requirement here or is there something in 2.10.4 that we need to depend on?